src/GCPClientManager.php

```diff
@@ -27,6 +26,18 @@ class GCPClientManager
     /** @var array<FoldersClient|ProjectsClient|ServiceUsageClient|AnalyticsHubServiceClient> */
     private array $clients = [];
 
+    public const DEFAULT_RETRY_SETTINGS =
+        [
+            // try max 3 times
+            'retries' => 3,
+            // multiplicator of backoff time between runs. First = $initial_delay ; second $previousDelay * $factor
+            'factor' => 1.1,
```

Comment:
hmm mohlo to byt lineární, ale je tam ten jitter, který to može posunůt před tů minutu takže asi cajk.